extras/pre-commit-hooks-extra.nix

```diff
@@ -0,0 +1,20 @@
+{ inputs, ... }: {
+  imports = [
+    inputs.pre-commit-hooks.flakeModule # Adds perSystem.pre-commit options
+  ];
+  perSystem = { pkgs, ... }:
+    let
+      rustfmt = pkgs.rust-bin.stable.latest.rustfmt;
+    in
+    {
+      pre-commit.settings.hooks = {
+        rustfmt-monorepo =
+          {
+            name = "rustfmt";
```

Comment:
Ended up with a ridiculously simple configuration, now that clippy is out of the picture